README.md

```diff
@@ -3,15 +3,22 @@
 ![Ui](docs/images/Ui.png)
 
 # Swift+
+### Manage your Team like never before
+#### Team: Swift+ (CS2103-T12-02)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
- 
+- Swift+ is a seamless contact management system to keep track of their interactions and meetings with clients and colleagues using swift text-based commands.
+- Swift+ allows you to plan your schedules looking at their daily/weekly overview. 
+
+## Getting Started
+1. Download and install `jenv` 
+2. Install `Java 11`
+3. Clone the project
+
+## Site Map
+- [User Guide](#)
+- [Developer Guide](#)
+- [About Us](#)
+- [Contact Us](#)
+
+## Acknowledgements
+- This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
```

Comment:
is this `AboutUs` page supposed to be edited in this PR?

Reply:
Let me edit this in the other PR, i made a mistake and forgot to make a branch for this feature